frontend/src/utils/wildcard.js

```diff
@@ -39,14 +41,32 @@ export function wildcardObjectsFromStrings (wildcardStrings) {
       endsWithWildcard,
       customWildcard,
       isWildcard: startsWithWildcard || endsWithWildcard || customWildcard,
-      regex: new RegExp('^' + pattern + '$')
+      test: function (value) {
+        if (!this.isWildcard) {
+          return value === this.originalValue
+        }
+        if (this.customWildcard) {
+          return true
+        }
+        if (this.startsWithWildcard && this.endsWithWildcard) {
+          return includes(value.substring(1, value.length - 1), this.value)
+        }
+        if (this.startsWithWildcard) {
+          return endsWith(value.substring(1, value.length), this.value)
+        }
+        if (this.endsWithWildcard) {
+          return startsWith(value.substring(0, value.length - 1), this.value)
+        }
```

Comment:
```suggestion
        if (this.startsWithWildcard && this.endsWithWildcard) {
          return includes(value, this.value)
        }
        if (this.startsWithWildcard) {
          return endsWith(value, this.value)
        }
        if (this.endsWithWildcard) {
          return startsWith(value, this.value)
        }
```
I am not the expert here but I think this is wrong.

Reply:
I'm also not the expert. I think this has been wrong in the previous implementation. At least we show the suffix / prefix as required on the UI